lib/handler.js

```diff
@@ -1,13 +1,10 @@
-"use strict" ;
```

Comment:
Why are we removing `"use strict" ;`? That's how we enable ES6 functionality, no?

Reply:
The linter says ` 'use strict' is unnecessary inside of modules`

Reply:
Getting a `'use strict' is unnecessary inside of modules` for these from eslint. What exactly does it have to do with enabling ES6 functionality?

Reply:
Interesting. Maybe I was only getting that error for specs originally.